assert/assertions.go

```diff
@@ -75,6 +81,20 @@ func ObjectsAreEqual(expected, actual interface{}) bool {
 	return bytes.Equal(exp, act)
 }
 
+func objectToTime(o interface{}) (time.Time, bool) {
+	s, ok := o.(time.Time)
+	if ok {
+		return s, ok
+	}
+
+	p, ok := o.(*time.Time)
+	if ok {
+		return *p, true
```

Comment:
`p` could be a nil pointer, please check that `p != nil` before dereferencing it.